docs/project/notes/music-terminology-removal-plan.md

```
@@ -0,0 +1,209 @@
+| Old Syntax                | New Syntax                 |
+| ------------------------- | -------------------------- |
+| `{{stem}}...{{/stem}}`    | `{{block}}...{{/block}}`   |
+| `{{stem name="example"}}` | `{{block name="example"}}` |
+| `{{> @mixin }}`           | `{{> @partial }}`          |
+| "stem content"            | "block content"            |
+| "stem name"               | "block name"               |
+
+## Implementation Checklist
+
+### Phase 1: Core Renaming
+
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)